go/internal/test_tools/test_tools.go

```diff
@@ -88,3 +88,14 @@ func GetPort() string {
 		return port
 	}
 }
+
+// GetAuth returns the auth string to connect to for the tests.
+// By default it is Anonymous, but can be overridden by setting the DH_AUTH environment variable.
+func GetAuth() string {
+	auth := os.Getenv("DH_AUTH")
+	if auth == "" {
+		return "Anonymous"
```

Comment:
Suggest moving magic string literals like this (and "http.session.durationMs") to some utility class so they can be defined in one place